io.catenax.remanufacturing_certificate/RELEASE_NOTES.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [3.0.0] - 2024-03-07
```

Comment:
update this date to 11th March in all your PRs if you want MS3 in next weekly meeting.

Reply:
updated